### DISCUSSION TITLE
Support for mmWave frequency

### DISCUSSION THREAD

**Discussion by liuqiang12040913** [NVlabs/sionna#210] [General]
Hi, I would like to know if sionna supports mmWave frequency. If yes, what configurations should be modified? If not, what may be the high-level approach to develop this module in sionna, if applicable?

**Comment by jmoraispk** [comment]
It does. Where you configure that would depend on what modules you are interested in using. Many are completely independent of the frequency. What are you thinking of using Sionna for?

**Reply by liuqiang12040913** [reply]
Thank you so much for the prompt reply. We plan to use Sionna for channel modeling in mmWave bands. To do so, you mean we can mostly change [this frequency](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html#:~:text=The%20default%20scenes,%5B8%5D%3A) to mmWave band, and the underlying mechanisms will just work (e.g., reflection, scattering)?

**Reply by jmoraispk** [reply]
**Exactly.** The ray tracing module fetches the frequency from `scene.frequency` whenever needed. 

For example, you can see [here](https://nvlabs.github.io/sionna/api/rt.html#radio-materials) that the radio materials have their properties dependent on the frequency, so they will automatically fit the frequency you choose. 

Please keep in mind the frequency ranges because some materials don't go up to mmWaves, but most do. And you can also define your own materials if needed.
Make sure to see also the [scattering tutorial](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Scattering.html), because the scattering coefficients are set to zero by default. They need to be set to a value different than zero if you want to see scattered paths with powers different than zero.

**Reply by jmoraispk** [reply]
@faycalaa does this sound about right?

**Reply by faycalaa** [reply]
Yes, that's correct.
